services/gundeck/test/integration/API.hs

```diff
@@ -181,7 +183,8 @@ replacePresence = do
     assertTrue "Cannon is not removed" $
       elem localhost8080 . map resource . decodePresence
   setPresence gu pres2
-    !!! const 201 === statusCode
+    !!! const 201
+    === statusCode
```

Comment:
if you start using some tool for curating commits, you can easily undo these things instead of commiting them.  [lots of options](https://git-scm.com/downloads/guis)  :)

Reply:
I am using [tig](https://github.com/jonas/tig) which is pretty cool.

I think, that if I revert the changes from auto-formatting the file, the linter will fail on CI.

Reply:
oh, ormolu did this?  interesting.  and mildly annoying.  :)  but we probably have talked to long about this already, sorry!